### PR TITLE
27 create classes for smoke testing

### PR DESCRIPTION
**Pull request by Cxristian**
update classes

### CODE REVIEW

File: src/main/java/org/fjala/gugumber/core/StrategySetter.java [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+ * with Jala Foundation.
  */
 
 package org.fjala.gugumber.core;
 
 /**
  * StrategySetter class.
  *
- * @author Cristian Lujan
+ * @author Cristian Lujan.
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/core/StrategySetter.java]: Don't use period after the @author name please

File: src/main/java/org/fjala/gugumber/salesforce/entities/Account.java [hunk 2]
```diff
@@ -67,7 +67,7 @@ public void setAccountInformation(final Map<String, String> accountMap) {
      * Returns a map with the information the account.
      *
      * @param accountMap to set the information.
-     * @return strateg ma with the information of account.
+     * @return strategy ma with the information of account.
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/entities/Account.java]: Review the comment please " @return strategy **ma** with the information of account."

File: src/main/java/org/fjala/gugumber/salesforce/keys/ContactKeys.java [hunk 3]
```diff
@@ -0,0 +1,161 @@
+package org.fjala.gugumber.salesforce.keys;
+
+/**
+ * ContactKeys class.
+ *
+ * @author Cristian Lujan.
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/keys/ContactKeys.java]: Clear the period after the @author name please

File: src/main/java/org/fjala/gugumber/salesforce/keys/ContactKeys.java [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+    /**
+     * Constant for key of Level.
+     */
+    public final static String LEVEL = "Level";
+
+    /**
+     * Constant for key of Description.
+     */
+    public final static String DESCRIPTION = "Description";
+
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/keys/ContactKeys.java]: Clear this line please

File: src/main/java/org/fjala/gugumber/salesforce/ui/PageLayoutFactory.java [hunk 5]
```diff
@@ -12,6 +12,7 @@
 
 package org.fjala.gugumber.salesforce.ui;
 
+
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/ui/PageLayoutFactory.java]: Clear this line please

File: src/main/java/org/fjala/gugumber/salesforce/ui/PageLayoutFactory.java [hunk 6]
```diff
@@ -21,6 +22,10 @@
 import org.fjala.gugumber.salesforce.ui.pages.app.BaseAppClassicPage;
 import org.fjala.gugumber.salesforce.ui.pages.app.BaseAppLightningPage;
 import org.fjala.gugumber.salesforce.ui.pages.app.BaseAppPage;
+import org.fjala.gugumber.salesforce.ui.pages.contact.ContactClassicPage;
+import org.fjala.gugumber.salesforce.ui.pages.contact.ContactLightningPage;
+import org.fjala.gugumber.salesforce.ui.pages.contact.ContactPageAbstract;
+
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/ui/PageLayoutFactory.java]: Clear this line please

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/components/NavBarClassic.java [hunk 7]
```diff
@@ -56,4 +81,5 @@ public AccountsPage clickAccountsOption() {
         accountOption.click();
         return new AccountsClassicPage();
     }
+
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/components/NavBarClassic.java]: Cleat this line please

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactClassicPage.java [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+    private WebElement phone1Txtb;
+
+    /**
+     * Locator for text box of Email.
+     */
+    @FindBy(id = "email")
+    private WebElement emailTxtb;
+
+    @FindBy(css = "th[class=' dataCell  ']")
+    private List<WebElement> contactNameList;
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactClassicPage.java]: Comment this attribute please

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactClassicProfilePage.java [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+     *
+     * @return new contact title.
+     */
+    @Override
+    public String getFullNameTitleContact() {
+        return contactTitle.getText();
+    }
+
+    @Override
+    public boolean isTheNewContact() {
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactClassicProfilePage.java]: Comment this method please

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactForm.java [hunk 10]
```diff
@@ -0,0 +1,330 @@
...
+import static org.fjala.gugumber.salesforce.keys.ContactKeys.SALUTATION;
+import static org.fjala.gugumber.salesforce.keys.ContactKeys.TITLE;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.fjala.gugumber.core.StrategySetter;
+import org.fjala.gugumber.salesforce.ui.pages.BasePage;
+
+
+
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactForm.java]: Clear this line please

Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactForm.java]: Clear this line please

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactForm.java [hunk 11]
```diff
@@ -0,0 +1,330 @@
...
+import static org.fjala.gugumber.salesforce.keys.ContactKeys.REPORTS_TO;
+import static org.fjala.gugumber.salesforce.keys.ContactKeys.SALUTATION;
+import static org.fjala.gugumber.salesforce.keys.ContactKeys.TITLE;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.fjala.gugumber.core.StrategySetter;
+import org.fjala.gugumber.salesforce.ui.pages.BasePage;
+
+
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactForm.java]: Clear this line please

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactForm.java [hunk 12]
```diff
@@ -0,0 +1,330 @@
...
+import static org.fjala.gugumber.salesforce.keys.ContactKeys.PHONE;
+import static org.fjala.gugumber.salesforce.keys.ContactKeys.REPORTS_TO;
+import static org.fjala.gugumber.salesforce.keys.ContactKeys.SALUTATION;
+import static org.fjala.gugumber.salesforce.keys.ContactKeys.TITLE;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.fjala.gugumber.core.StrategySetter;
+import org.fjala.gugumber.salesforce.ui.pages.BasePage;
+
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactForm.java]: Clear this line please

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactForm.java [hunk 13]
```diff
@@ -0,0 +1,330 @@
...
+        strategyMap.put(SALUTATION,() -> setSalutation(contact.get(SALUTATION)));
+        strategyMap.put(FIRST_NAME,() -> setFirstName(contact.get(FIRST_NAME)));
+        strategyMap.put(LAST_NAME,() -> setLastName(contact.get(LAST_NAME)));
+        strategyMap.put(ACCOUNT,() -> setAccount(contact.get(ACCOUNT)));
+        strategyMap.put(PHONE,() -> setPhone(Integer.parseInt(contact.get(PHONE))));
+        strategyMap.put(EMAIL,() -> setEmail(contact.get(EMAIL)));
+        strategyMap.put(HOME_PHONE,() -> setHomePhone(Integer.parseInt(contact.get(HOME_PHONE))));
+        strategyMap.put(TITLE,() -> setTitle(contact.get(TITLE)));
+        strategyMap.put(DEPARTMENT,() -> setDepartment(contact.get(DEPARTMENT)));
+//        strategyMap.put(BIRTHDATE,() -> setBirthdate(newContact.get(BIRTHDATE)));
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactForm.java]: Clear this comment please

File: src/main/java/org/fjala/gugumber/core/selenium/WebDriverConfig.java [hunk 14]
```diff
@@ -23,37 +23,37 @@
...
 
     /**
-     * Constant for the sleep wait  time.
+     * Constant for the sleep wait time.
      */
     private static final String WAIT_SLEEP_TIME = "waitSleepTime";
 
     /**
-     * Implicit wait time saves the implicit wait time.
+     * Variable to implicit wait time.
```
Comment (AKatrin) [inline src/main/java/org/fjala/gugumber/core/selenium/WebDriverConfig.java]: chand the "Variable to" to "Varibale for"

File: src/main/java/org/fjala/gugumber/core/selenium/utils/DriverMethods.java [hunk 15]
```diff
@@ -33,10 +35,21 @@ public static void setTxt(final WebElement webElement, final String text) {
...
+        webElement.click();
+        driver.findElement(By.cssSelector(replaceCmb.replace("nameTitle", text))).click();
+    }
 
     //TODO method to clear a checkbox
 
     //TODO method que valida que un elemento este en el DOM cambiando el implicit wait
     //TODO validate method that a webElement is in the DOM change the implicit wait
-}
+}
```
Comment (AKatrin) [inline src/main/java/org/fjala/gugumber/core/selenium/utils/DriverMethods.java]: Add a line end

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactLightningProfilePage.java [hunk 16]
```diff
@@ -0,0 +1,84 @@
...
+ * This software is the confidential and proprietary information of Jala
+ * Foundation, Inc. ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package org.fjala.gugumber.salesforce.ui.pages.contact;
+
+
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactLightningProfilePage.java]: Clear this line please

File: src/main/java/org/fjala/gugumber/salesforce/entities/Context.java [hunk 17]
```diff
@@ -21,7 +21,7 @@
 public class Context {
 
     /**
-     * Contact of type contact.
+     * contact of type contact.
```
Comment (AKatrin) [inline src/main/java/org/fjala/gugumber/salesforce/entities/Context.java]: The documentation start with uppercase

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactLightningProfilePage.java [hunk 18]
```diff
@@ -0,0 +1,84 @@
...
+     * @return new contact title.
+     */
+    @Override
+    public String getFullNameTitleContact() {
+        return contactTitle.getText();
+    }
+
+
+    @Override
+    public boolean isTheNewContact() {
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactLightningProfilePage.java]: Comment this method please

File: src/main/java/org/fjala/gugumber/salesforce/keys/ContactKeys.java (hunk 4 above)
Comment (AKatrin) [reply src/main/java/org/fjala/gugumber/salesforce/keys/ContactKeys.java]: Add the constructor please

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactForm.java (hunk 13 above)
Comment (AKatrin) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactForm.java]: uncomment this line or remove

File: src/test/java/org/fjala/gugumber/hooks/Hooks.java [hunk 19]
```diff
@@ -0,0 +1,53 @@
...
+                byte[] screenshot = ((TakesScreenshot) driver).getScreenshotAs(OutputType.BYTES);
+                scenario.embed(screenshot, "image/png");
+            } catch (WebDriverException wde) {
+                System.err.println(wde.getMessage());
+            } catch (ClassCastException cce) {
+                cce.printStackTrace();
+            }
+        }
+    }
+}
```
Comment (AKatrin) [inline src/test/java/org/fjala/gugumber/hooks/Hooks.java]: add end line

File: src/test/java/org/fjala/gugumber/steps/ContactSteps.java [hunk 20]
```diff
@@ -0,0 +1,103 @@
...
+    private Context context;
+    /**
+     * Variable for the contact.
+     */
+    private Contact contact;
+
+    private PageLayoutType layout = PageLayoutConfig.getPageLayoutName();
+
+    //pages
+    private ContactPageAbstract contactPage;
```
Comment (AKatrin) [inline src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: Add documentation please

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactPageAbstract.java [hunk 21]
```diff
@@ -0,0 +1,40 @@
...
+     */
+    public abstract ContactForm clickNewContact();
+
+    /**
+     * Returns a list with last name from the contact page.
+     *
+     * @return a list of last names contact as string.
+     */
+    public abstract List<String> getListOfContactsName();
+}
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactPageAbstract.java]: Add a blank line below this line please

File: src/test/java/org/fjala/gugumber/steps/ContactSteps.java [hunk 22]
```diff
@@ -0,0 +1,103 @@
...
+     * Constructor of contact steps sending the context.
+     *
+     * @param context init the context.
+     */
+    public ContactSteps(Context context) {
+        this.context = context;
+        contact = context.getContact();
+    }
+
+    @When("^I open Contact form$")
```
Comment (AKatrin) [inline src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: Add documentation please

File: src/test/java/org/fjala/gugumber/steps/LoginSteps.java [hunk 23]
```diff
@@ -26,10 +26,13 @@
     private PageTransporter pageTransporter = PageTransporter.getInstance();
     private LoginPage loginPage;
 
+
```
Comment (AKatrin) [inline src/test/java/org/fjala/gugumber/steps/LoginSteps.java]: Add documentation please

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactProfilePage.java [hunk 24]
```diff
@@ -0,0 +1,38 @@
...
+     */
+    public abstract String getFullNameTitleContact();
+
+    /**
+     * Returns the icon of contact from a header profile.
+     *
+     * @return the icon of contact.
+     */
+    public abstract boolean isTheNewContact();
+}
```
Comment (raultolive) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactProfilePage.java]: Add a blank line below this line please

File: src/test/resources/features/event.feature [hunk 25]
```diff
@@ -1,38 +1,38 @@
...
-    When  I open the new Event
-    Then  the data of new Event should be displayed from Event Detail page
-
-  @createEvent
-  Scenario Delete an Event
-    When  I open the "Other" Event
-    And   I delete the Event
-    Then  the Event should be clear from the list of Events
\ No newline at end of file
+#Feature: Event manage.
```
Comment (AKatrin) [inline src/test/resources/features/event.feature]: uncomment or remove this

File: src/test/java/org/fjala/gugumber/hooks/Hooks.java [hunk 26]
```diff
@@ -0,0 +1,53 @@
...
+
+/**
+ * Hooks class
+ *
+ * @author Cristian Lujan
+ * @version 0.0.1
+ */
+public class Hooks {
+
+    private final Context context;
```
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/hooks/Hooks.java]: Comment this attribute please

File: src/test/java/org/fjala/gugumber/hooks/Hooks.java [hunk 27]
```diff
@@ -0,0 +1,53 @@
...
+    private WebDriver driver;
```
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/hooks/Hooks.java]: Comment this attribute please

File: src/test/java/org/fjala/gugumber/hooks/Hooks.java (hunk 19 above)
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/hooks/Hooks.java]: Add a blank line below this line please

File: src/test/java/org/fjala/gugumber/runner/RunCukesTest.java [hunk 28]
```diff
@@ -37,7 +37,7 @@
      */
     @AfterTest
     public void afterExecution() {
-        WebDriverManager.getInstance().getWebDriver().close();
+//        WebDriverManager.getInstance().getWebDriver().close();
```
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/runner/RunCukesTest.java]: Clear or uncomment this line please

File: src/test/java/org/fjala/gugumber/steps/ContactSteps.java [hunk 29]
```diff
@@ -0,0 +1,103 @@
...
+ * @author Cristian Lujan
+ * @version 0.0.1
+ */
+public class ContactSteps {
+
+    /**
+     * Variable for the context.
+     */
+    private Context context;
+    /**
```
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: Add a blank above this line please

File: src/test/java/org/fjala/gugumber/steps/ContactSteps.java [hunk 30]
```diff
@@ -0,0 +1,103 @@
...
+            assertEquals(message, "Contact was created.", "not successfully created");
+        } else {
+            Logs.getInstance().getLog().info("In classic Layout is not message of confirmation");
+        }
+    }
+
+    @Then("^the contact last name should be displayed in the Contact Profile page$")
+    public void theContactLastNameShouldBeDisplayedInTheContactProfilePage() {
+        assertEquals(contactProfilePage.getFullNameTitleContact(), contact.getFullName(),"the Contact Last name not displayed");
+//        assertTrue(contactProfilePage.isTheNewContact());
```
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: Clear or uncomment this line please

File: src/test/java/org/fjala/gugumber/steps/ContactSteps.java [hunk 31]
```diff
@@ -0,0 +1,103 @@
...
+    /**
+     * Variable for the context.
+     */
+    private Context context;
+    /**
+     * Variable for the contact.
+     */
+    private Contact contact;
+
+    private PageLayoutType layout = PageLayoutConfig.getPageLayoutName();
```
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: Comment this attribute please

File: src/test/java/org/fjala/gugumber/steps/ContactSteps.java [hunk 32]
```diff
@@ -0,0 +1,103 @@
...
+
+    //pages
+    private ContactPageAbstract contactPage;
+    private ContactForm contactForm;
+    private ContactProfilePage contactProfilePage;
```
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: Comment this attribute please

File: src/test/java/org/fjala/gugumber/steps/ContactSteps.java [hunk 33]
```diff
@@ -0,0 +1,103 @@
...
+    /**
+     * Variable for the contact.
+     */
+    private Contact contact;
+
+    private PageLayoutType layout = PageLayoutConfig.getPageLayoutName();
+
+    //pages
+    private ContactPageAbstract contactPage;
+    private ContactForm contactForm;
```
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: Comment this attribute please

Comment (raultolive) [inline src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: Comment this attribute please

File: src/test/java/org/fjala/gugumber/steps/ContactSteps.java (hunk 20 above)
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: Comment this attribute please

File: src/test/java/org/fjala/gugumber/steps/ContactSteps.java [hunk 34]
```diff
@@ -0,0 +1,103 @@
...
+     *
+     * @param context init the context.
+     */
+    public ContactSteps(Context context) {
+        this.context = context;
+        contact = context.getContact();
+    }
+
+    @When("^I open Contact form$")
+    public void OpenContactForm() {
```
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: Comment this method please

File: src/test/java/org/fjala/gugumber/steps/ContactSteps.java [hunk 35]
```diff
@@ -0,0 +1,103 @@
...
+        contactPage = PageLayoutFactory.getContactPage();
+        contactForm = contactPage.clickNewContact();
+    }
+
+    @And("^I create a new Contact with the following information in Contact form$")
+    public void CreateANewContactInContactForm(Map<String, String> contactMap) {
```
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: Comment this method please

File: src/test/java/org/fjala/gugumber/steps/ContactSteps.java [hunk 36]
```diff
@@ -0,0 +1,103 @@
...
+        contact.processInformation(contactMap);
+        contactForm.setContactInformation(contactMap);
+        contactProfilePage = contactForm.clickSaveNewContact();
+    }
+
+    @Then("^a message that indicates the Contact was created should be displayed$")
+    public void MessageThatIndicatesTheContactWasCreated() {
```
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: Comment this method please

File: src/test/java/org/fjala/gugumber/steps/ContactSteps.java [hunk 37]
```diff
@@ -0,0 +1,103 @@
...
+            ContactLightningProfilePage profileContactLightningPage = new ContactLightningProfilePage();
+            final String message = (profileContactLightningPage.getMessageSave());
+            assertEquals(message, "Contact was created.", "not successfully created");
+        } else {
+            Logs.getInstance().getLog().info("In classic Layout is not message of confirmation");
+        }
+    }
+
+    @Then("^the contact last name should be displayed in the Contact Profile page$")
+    public void theContactLastNameShouldBeDisplayedInTheContactProfilePage() {
```
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: Comment this method please

File: src/test/java/org/fjala/gugumber/steps/ContactSteps.java (hunk 30 above)
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: Clear or uncomment this line please

File: src/test/java/org/fjala/gugumber/steps/ContactSteps.java [hunk 38]
```diff
@@ -0,0 +1,103 @@
...
+        assertEquals(contactProfilePage.getFullNameTitleContact(), contact.getFullName(),"the Contact Last name not displayed");
+//        assertTrue(contactProfilePage.isTheNewContact());
+    }
+
+    @Then("^the contact last name should be displayed in the contacts list of Contacts page$")
+    public void theContactLastNameShouldBeDisplayedInTheContactsListOfContactsPage() {
```
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: Comment this method please

File: src/test/java/org/fjala/gugumber/steps/LoginSteps.java [hunk 39]
```diff
@@ -26,10 +26,13 @@
...
     @Given("^I am logged in SalesForce with \"([^\"]*)\" and \"([^\"]*)\"$")
     public void logInWithUsernameAdminAndPasswordAdmin(String username, String password) {
```
Comment (raultolive) [inline src/test/java/org/fjala/gugumber/steps/LoginSteps.java]: Comment this method please

File: src/test/resources/features/event.feature [hunk 40]
```diff
@@ -1,38 +1,38 @@
...
+#      |Description|Description Text        |
+#    Then  the Subject of new Event should be displayed on Calendar Section
+#    When  I open the new Event
+#    Then  the data of new Event should be displayed from Event Detail page
+#
+#  @createEvent
+#  Scenario Delete an Event
+#    When  I open the "Other" Event
+#    And   I delete the Event
+#    Then  the Event should be clear from the list of Events
```
Comment (raultolive) [inline src/test/resources/features/event.feature]: Add a blank line below this line please

File: src/test/resources/features/event.feature (hunk 25 above)
Comment (raultolive) [inline src/test/resources/features/event.feature]: Clear or uncomment this attribute please

File: src/test/resources/features/login.feature [hunk 41]
```diff
@@ -1,4 +1,4 @@
-Feature: Log in SalesForce
-  Scenario: Login in SalesForce with valid user account
-    Given I am logged in SalesForce with "username.admin" and "password.admin"
-    When I navigate to the Home page
\ No newline at end of file
+#Feature: Log in SalesForce
```
Comment (raultolive) [inline src/test/resources/features/login.feature]: Clear or uncomment this line please

File: src/test/resources/features/login.feature [hunk 42]
```diff
@@ -1,4 +1,4 @@
...
+#  Scenario: Login in SalesForce with valid user account
+#    Given I am logged in SalesForce with "username.admin" and "password.admin"
+#    When I navigate to the Home page
```
Comment (raultolive) [inline src/test/resources/features/login.feature]: Add a blank line below this line please

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactForm.java (hunk 13 above)
Comment (Cxristian) [reply src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactForm.java]: at this moment it has to be commented only that line

File: src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactLightningPage.java [hunk 43]
```diff
@@ -0,0 +1,95 @@
...
+    @Override
+    public List<String> getListOfContactsName() {
+        final List<String> contactName = new ArrayList<>();
+        for (WebElement contName : contactNameList) {
+            contactName.add(contName.getText());
+        }
+        return contactName;
+    }
+
+    public void openContactProfile(final String text) {
```
Comment (AKatrin) [inline src/main/java/org/fjala/gugumber/salesforce/ui/pages/contact/ContactLightningPage.java]: Add documentation

File: src/test/java/org/fjala/gugumber/steps/BaseAppSteps.java [hunk 44]
```diff
@@ -35,6 +41,15 @@
      */
     private BaseAppPage baseAppPage;
 
+    /**
+     * Opens the contacts page.
+     */
+    @When("^I go to the Contacts page$")
+    public void GoToTheContactsPage() {
```
Comment (AKatrin) [inline src/test/java/org/fjala/gugumber/steps/BaseAppSteps.java]: the method should start with lower case

File: src/test/java/org/fjala/gugumber/steps/ContactSteps.java (hunk 34 above)
Comment (AKatrin) [reply src/test/java/org/fjala/gugumber/steps/ContactSteps.java]: The method should start with lowercase

**Pull request merged**